custom/src/test/java/co/elastic/otel/ElasticAutoConfigurationCustomizerProviderTest.java

```diff
@@ -67,4 +67,13 @@ void disableExperimentalRuntimeMetrics() {
     String value = config.get("otel.instrumentation.runtime-telemetry.emit-experimental-telemetry");
     assertThat(value).isEqualTo("false");
   }
+
+  @Test
+  void customizeMetricTemporalityPreference() {
```

Comment:
[minor] probably worth adding a test to ensure we override the default.

Reply:
Fixed in [8790acb](https://github.com/elastic/elastic-otel-java/pull/583/commits/8790acb2a44a26b200e2eb70f415161dac86b0f3)